integration/spark/src/main/spark2.4/org/apache/spark/sql/CarbonToSparkAdapter.scala

```diff
@@ -193,9 +193,17 @@ class CarbonOptimizer(
     catalog: SessionCatalog,
     optimizer: Optimizer) extends Optimizer(catalog) {
 
-  private lazy val mvRules = Seq(Batch("Materialized View Optimizers", Once,
+  private lazy val _mvRules = Seq(Batch("Materialized View Optimizers", Once,
     Seq(new MVRewriteRule(session)): _*))
 
+  private def mvRules: Seq[Batch] = {
+    // enable mv by default
+    session.conf.get("spark.carbon.mv.enable", "true").toBoolean match {
```

Comment:
make a  constant for default value

Reply:
this is a runtime configuration.
so the user can use the SET command to change it.

Reply:
conf name should start with carbon, and please add in doc also